nginx/templates/default.conf.jinja

```diff
@@ -30,8 +31,11 @@ server {
   root {{ webroot }};
   server_name {{ domain }} www.{{ domain }} {{ aliases|default([])|join(" ") }};
 
+  {{- before_location_config|default([])|join("\n")|indent(2) }}
+
   location / {
       try_files {{ try_files|default('$uri $uri/ /index.php?$args') }};
+      {{- extra_location_config|default([])|join("\n")|indent(4) }}
```

Comment:
aah, this one confuses me :thinking: 

`extra_location_config` doesn't seem to cover what your enabling here. Since you're adding configuration options for the root location block. Might be cool to rename it to something like `root_location_config` and then let the user either overwrite the entire block or keep the default `try_files` setup. I'm suggesting this because you might want to configure a proxy_pass here instead of a try files.

Reply:
It's an extension to the idea of `extra_config`, and it is is in the location area. So that's why I picked this name.